### PR TITLE
Remove fast fail

### PR DESCRIPTION
**Pull request by Emiller88**
Per slack conversation with @ewels 
https://nfcore.slack.com/archives/CP3RJSMF0/p1671232975539149

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

**Review by ewels — APPROVED** [review]
Yay, only _partial_ CI failures! 🚀 

image

**Comment by Emiller88** [comment]
@ewels I don't have permissions to merge(maybe at all?) without the passing CI, so can you push this one through?

**Comment by ewels** [comment]
@Emiller88 can you configure the linting to ignore the relevant test with a `TODO` comment to remove when the template is updated?

Then we should restart the other failing tests until they pass (which is what this PR makes easier I hope!)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
     if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/methylseq') }}"
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```
Comment (sateeshperi) [inline .github/workflows/ci.yml:25]: @ewels do we still want this fail fast to false in the new CI ?

Comment (sateeshperi) [reply .github/workflows/ci.yml:25]: looks like it will be handy to not kill all tests at once...so going to add this in

**Comment by sateeshperi** [comment]
caching of test-data implemented in new CI.yml. closing this now

**Pull request closed without merging**